test/utils/test_image_utils.py

```diff
@@ -0,0 +1,38 @@
+import unittest
```

Comment:
No action required, but as you have two tests, these could be split to two pull requests